### PR TITLE
tag volumes with the provider tags

### PR DESCRIPTION
**Pull request by tzneal**
Pass provider tags to CreateFleet for volumes as well.

**1. Issue, if available:**
#1430

**2. Description of changes:**

**3. How was this change tested?**

Configured a provisioner with tags and inspected the volumes created in the EC2 console to ensure that the tags were applied. 

**4. Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [X] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/cloudprovider/aws/instance.go [hunk 1]
```diff
@@ -127,6 +127,10 @@ func (p *InstanceProvider) launchInstances(ctx context.Context, constraints *v1a
 				ResourceType: aws.String(ec2.ResourceTypeInstance),
 				Tags:         v1alpha1.MergeTags(ctx, constraints.Tags, map[string]string{fmt.Sprintf("kubernetes.io/cluster/%s", injection.GetOptions(ctx).ClusterName): "owned"}),
 			},
+			{
+				ResourceType: aws.String(ec2.ResourceTypeVolume),
+				Tags:         v1alpha1.MergeTags(ctx, constraints.Tags, map[string]string{fmt.Sprintf("kubernetes.io/cluster/%s", injection.GetOptions(ctx).ClusterName): "owned"}),
```
Comment (ellistarn) [inline pkg/cloudprovider/aws/instance.go:132]: Thoughts on extracting out the merge tags call? This has a nice side effect of shortening the line enough that you can collapse the lines elsewhere:
```
TagSpecifications: []*ec2.TagSpecification{
    { ResourceType: aws.String(ec2.ResourceTypeInstance), Tags: tags },
    { ResourceType: aws.String(ec2. ResourceTypeVolume), Tags: tags },
}
```

**Review by ellistarn — APPROVED** [review]
Nice! One optional comment.

File: pkg/cloudprovider/aws/instance.go (hunk 1 above)
Comment (tzneal) [reply pkg/cloudprovider/aws/instance.go:132]: LGTM

**Review by spring1843 — DISMISSED** [review]

**Comment by tzneal** [comment]
Incorporated feedback and re-tested live.

**Review by ellistarn — APPROVED** [review]

**Comment by ellistarn** [comment]
Nice feature! Thank you! Can you follow up with a documentation change?

**Pull request merged**